src/main/msp/msp_protocol.h

```diff
@@ -62,7 +62,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1  // increment when major changes are made
-#define API_VERSION_MINOR                   45 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
+#define API_VERSION_MINOR                   46 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
```

Comment:
I guess you want to avoid rebasing this part once BF 4.4 gets released?

Reply:
I was assuming this wouldn't make it into 4.4 given that we're past the code freeze.